sortedm2m/fields.py

```diff
@@ -171,13 +177,31 @@ def _add_items(self, source_field_name, target_field_name, *objs):
     return SortedRelatedManager
 
 
-class ReverseSortedManyRelatedObjectsDescriptor(ReverseManyRelatedObjectsDescriptor):
-    @property
-    def related_manager_cls(self):
-        return create_sorted_many_related_manager(
-            self.field.rel.to._default_manager.__class__,
-            self.field.rel
-        )
+try:
+    from django.db.models.fields.related import ReverseManyRelatedObjectsDescriptor
+
+    class ReverseSortedManyRelatedObjectsDescriptor(ReverseManyRelatedObjectsDescriptor):
+        @cached_property
+        def related_manager_cls(self):
+            return create_sorted_many_related_manager(
+                self.field.rel.to._default_manager.__class__,
+                self.field.rel
+            )
+except ImportError:
+    from django.db.models.fields.related import ManyRelatedObjectsDescriptor
+
+    # ReverseManyRelatedObjectsDescriptor was removed from Django 1.9
+    class ReverseSortedManyRelatedObjectsDescriptor(ManyRelatedObjectsDescriptor):
+        def __init__(self, field):
+            super(ReverseSortedManyRelatedObjectsDescriptor, self).__init__(field.remote_field)
+
+        @cached_property
+        def related_manager_cls(self):
+            return create_sorted_many_related_manager(
+                self.rel.model._default_manager.__class__,
+                self.rel,
+                False  # This is the new `reverse` argument (which ironically should be False)
```

Comment:
Could we use keyword arguments here, so that it's clear (even without the comment) what the arguments stand for?